metals/src/main/scala/scala/meta/internal/metals/ClientExperimentalCapabilities.scala

```diff
@@ -3,18 +3,23 @@ import com.google.gson.JsonElement
 import org.eclipse.{lsp4j => l}
 
 final case class ClientExperimentalCapabilities(
-    debuggingProvider: java.lang.Boolean = false,
-    treeViewProvider: java.lang.Boolean = false,
-    decorationProvider: java.lang.Boolean = false,
-    inputBoxProvider: java.lang.Boolean = false,
-    quickPickProvider: java.lang.Boolean = false,
-    didFocusProvider: java.lang.Boolean = false,
-    slowTaskProvider: java.lang.Boolean = false,
-    executeClientCommandProvider: java.lang.Boolean = false,
-    openFilesOnRenameProvider: java.lang.Boolean = false,
-    doctorProvider: String = "html",
-    statusBarProvider: String = "off"
+    debuggingProvider: Boolean,
+    decorationProvider: Boolean,
+    didFocusProvider: Boolean,
+    doctorProvider: String,
+    executeClientCommandProvider: Boolean,
+    inputBoxProvider: Boolean,
+    openFilesOnRenameProvider: Boolean,
+    quickPickProvider: Boolean,
+    slowTaskProvider: Boolean,
+    statusBarProvider: String,
+    treeViewProvider: Boolean
 ) {
+  def this() =
+    this(
+      false, false, false, "html", false, false, false, false, false, "off",
```

Comment:
they are all false so not hard to recognize what is what, but could we name them? decorationProvider = false , etc

Reply:
Ah totally. I meant to do this and forgot 👍